README

```diff
@@ -1 +1,2 @@
-This is a repo to practice Git
+This is a repo to practice Git.
+I am Testing for git commands.
```

Comment:
Please remove this line, just have the period above

Reply:
Okay.